Sources/X509/Verifier/PolicyBuilder.swift

```diff
@@ -111,10 +111,13 @@ extension PolicyBuilder {
     struct WrappedOptional<Wrapped>: VerifierPolicy where Wrapped: VerifierPolicy {
         @usableFromInline
         var wrapped: Wrapped?
+        @usableFromInline
+        var defaultResult: PolicyEvaluationResult
```

Comment:
I would prefer two separate implementations that have the default result hardcoded. Policies are already getting really large as they compose into a single inline memory allocation and we don't need this dynamic behaviour.